client/classic/upe/legacy-support.js

```diff
@@ -0,0 +1,69 @@
+import $ from 'jquery';
+import { getStripeServerData } from 'wcstripe/stripe-utils';
+
+/**
+ * Handles hashchange events when using PRBs.
+ *
+ * @param {Object}   api        The Stripe API interface.
+ * @param {Function} showError  Function used to present an error to the customer.
+ */
+export const legacyHashchangeHandler = ( api, showError ) => {
```

Comment:
This is basically a copy of https://github.com/woocommerce/woocommerce-gateway-stripe/blob/8eba0d6176c40107c12c2ebd0cfe362549bbb18d/assets/js/stripe.js#L799-L814